app/models/application_data.rb

```diff
@@ -0,0 +1,265 @@
+class ApplicationData
+  def initialize(data)
+    @data = data || {}
+  end
+
+  def student0_application_gender_identification
+    @data["student0_application_gender_identification"]
```

Comment:
Also a personal habit of mine with no impact on the functionality or visibility: I'd like to define a private `attr_reader` for things that get assigned within the constructor and then spare all the `@`s afterwards.

But as I said, just a personal habit of mine, no need to adapt this here.